tests/integration/test_orders_api.py

```diff
@@ -149,26 +149,36 @@ async def test_list_orders_state_invalid_state(session):
 
 @respx.mock
 @pytest.mark.asyncio
-async def test_list_orders_limit(order_descriptions, session):
-    nono_page_url = TEST_ORDERS_URL + '?page_marker=OhNoNo'
+@pytest.mark.parametrize("limit,limitted_list_length", [(None, 100), (0, 102),
```

Comment:
Nit: limited instead of limitted.

Reply:
IDEs need to also have spell check lol - good find, thanks!